### DISCUSSION TITLE
Трансляция видео напрямую на матрицу

### DISCUSSION THREAD

**Discussion by HuskySmile** [vvip-68/LedPanelWiFi#17] [Вопросы и ответы]
Добрый вечер. Пытаюсь найти информацию как транслировать видео напрямую на матрицу, но ссылка оставленная в этой теме

Настройка внешнего источника синхронизации на примере программы управления световым оборудованием "Jinx!" описана в этой [статье](https://github.com/vvip-68/GyverPanelWiFi/wiki/%D0%9D%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-%D0%9F%D0%9E-%D1%83%D0%BF%D1%80%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F-%D1%8D%D1%84%D1%84%D0%B5%D0%BA%D1%82%D0%B0%D0%BC%D0%B8-%22Jinx!%22-%D0%BA%D0%B0%D0%BA-%D0%B8%D1%81%D1%82%D0%BE%D1%87%D0%BD%D0%B8%D0%BA%D0%B0-%D1%81%D0%B8%D0%BD%D1%85%D1%80%D0%BE%D0%BD%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D0%B8-%D1%83%D1%81%D1%82%D1%80%D0%BE%D0%B9%D1%81%D1%82%D0%B2-%D0%B2-%D0%B3%D1%80%D1%83%D0%BF%D0%BF%D0%B5).

ведет на главную страницу. Может я тупо найти не могу в ней эту тему.

Подскажите пожалуйста в каком посте есть инструкция по настройке Jinx для трансляции видео на матрице в режиме реального времени

**Comment by vvip-68** [comment]
https://github.com/vvip-68/LedPanelWiFi/wiki/Настройка-ПО-управления-эффектами-Jinx!-как-источника-синхронизации-устройств-в-группе

![image](https://github.com/vvip-68/LedPanelWiFi/assets/39353139/87a4ad97-aed0-4892-a65d-728d427d3d69)

**Reply by huskydogsmile** [reply]
Спасибо! Все получилось, телек нормальный получился. Правда фпс ~20, но это уже особенности платы наверное...

**Reply by vvip-68** [reply]
Мне кажется Джинкс так отдает... Я в свое время искал, регулировки скорости выдачи кадров у него не нашел. Ставил отладку на код трансляции кадров из Джинкса в скетче - пропуска кадров не было. Все что джинкс (и с какой скоростью) отдавал - все показывалось.
